symfony/plugins/orangehrmLeavePlugin/Dao/LeaveEntitlementDao.php

```diff
@@ -99,6 +99,8 @@ private function getLeaveEntitlementsQueryBuilderWrapper(
         $q = $this->createQueryBuilder(LeaveEntitlement::class, 'entitlement')
             ->andWhere('entitlement.deleted = :deleted')
             ->setParameter('deleted', false);
+        $q->leftJoin('entitlement.employee', 'employee')
+            ->andWhere($q->expr()->isNull('employee.purgedAt'));
```

Comment:
**Necessary?**

Reply:
isn't this affecting in employee leave entitlement screen?

Reply:
I added this because of some failing test cases. I remove it and changed the test cases.